browser/main/Detail/MarkdownNoteDetail.js

```diff
@@ -51,22 +51,23 @@ class MarkdownNoteDetail extends React.Component {
       isLockButtonShown: props.config.editor.type !== 'SPLIT',
       isLocked: false,
       editorType: props.config.editor.type,
-      switchPreview: props.config.editor.switchPreview,
-      RTL: false
```

Comment:
Please don't remove this `RTL` key, this is for Right To Left feature, people who use this feature will be very angry :smile: